packages/playground/src/components/account/account-register/account-register.tsx

```diff
@@ -198,14 +203,19 @@ export class AccountRegister {
             error={this.errors.email}
             onChange={e => this.change("email", e)}
           />
-          <form-input
+          {/* <form-input
```

Comment:
@snario Why are we hiding the current address? Seems to be useful information. We might render it like something else instead of a text input, i.e. a label.

Reply:
Yeah I think I'll render it differently, but it doesn't make sense to be a form element